### PR TITLE
chore: Update oidc commands to include region and add brackets for trust policy

### PR DESCRIPTION
**Pull request by ryansteakley**
**Description of your changes:**

- Trust policy creation command needs {} to work properly and added --region to the describe cluster command.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by rrrkharse — APPROVED** [review]

File: samples/contrib/aws-samples/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ There are two ways you can give them access to SageMaker.
       ```
    2. Take note of the OIDC issuer URL. This URL is in the form `oidc.eks.<region>.amazonaws.com/id/<OIDC_ID>` . Note down the URL.
       ```
-      aws eks describe-cluster --name <cluster_name> --query "cluster.identity.oidc.issuer" --output text
+      aws eks describe-cluster --name <cluster_name> --region <cluster-region> --query "cluster.identity.oidc.issuer" --output text
```
Comment (mbaijal) [inline samples/contrib/aws-samples/README.md:33]: Eventually we should export these as variables instead

**Review by mbaijal — APPROVED** [review]

**Pull request merged**